vlasovsolver/vlasovmover.cpp

```diff
@@ -455,73 +437,70 @@ void calculateAcceleration(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>&
       for (uint popID=0; popID<getObjectWrapper().particleSpecies.size(); ++popID) {
          adjustVelocityBlocks(mpiGrid, cells, true, popID);
       }
+   } else {
+      // Fairly ugly but no goto
```

Comment:
This part is the only actual change of code semantics, other than phiprof variable reordering.
The testpackage result shows that this is working all as intended, but was this change required for the timer objects, or was it just something that irked you and you wanted to go away?

Reply:
iirc that actually didn't compile with the goto since it was jumping over a variable declaration

Reply:
Fair enough, that's definitely a sufficient reason to restructure.